src/components/stepper/index.tsx

```diff
@@ -173,6 +177,7 @@ class Stepper extends PureComponent<Props, State> {
     const allowStepChange = this.allowStepChange(actionType);
     const minusButton = small ? minusOutlineSmall : minusOutline;
     const plusButton = small ? plusOutlineSmall : plusOutline;
+    const useCustomTheme = this.props?.useCustomTheme;
```

Comment:
Why do you need keep a const for useCustomTheme?

Reply:
Instead should I pass the the useCsutomValue prop in the Button component?
`
<Button . . . useCustomValue = {this.props?.useCustomTheme} /> 
`
The const is just to have the value from the prop.
Is that a bad practice?